smartsim/_core/control/controller.py

```diff
@@ -104,12 +119,19 @@ def start(
         self._jobs.kill_on_interrupt = kill_on_interrupt
         # register custom signal handler for ^C (SIGINT)
         signal.signal(signal.SIGINT, self._jobs.signal_interrupt)
-        self._launch(manifest)
+        launched = self._launch(exp_name, exp_path, manifest)
 
         # start the job manager thread if not already started
         if not self._jobs.actively_monitoring:
             self._jobs.start()
 
+        serialize.save_launch_manifest(
+            launched.map(_look_up_launched_data(self._launcher))
+        )
+
+        # launch a telemetry monitor to track job progress
+        self.start_telemetry_monitor(exp_path)
```

Comment:
`start_telemetry_monitor` checks whether telemetry is enabled or not. Move this check up here, otherwise a plain reading makes it seem that telemetry is enabled unconditionally

Reply:
Good call! I think original idea was to limit the number of `if telemeter monitor enabled` touch points, but if it is only used here, it seems reasonable to move the check here (at the very least I agree it becomes more apparent what the control flow is).